library/src/main/java/com/schibsted/spain/barista/internal/util/DrawableToBitmapConverter.kt

```diff
@@ -0,0 +1,17 @@
+package com.schibsted.spain.barista.internal.util
+
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.drawable.Drawable
+
+internal class DrawableToBitmapConverter {
+  companion object {
+    fun getBitmap(drawable: Drawable?): Bitmap {
```

Comment:
You can reduce this method to:
```kotlin
fun getBitmap(drawable: Drawable?) = Bitmap.createBitmap(drawable!!.intrinsicWidth, drawable.intrinsicHeight, Bitmap.Config.ARGB_8888)
    .apply {
      val canvas = Canvas(this)
      drawable.setBounds(0, 0, canvas.width, canvas.height)
      drawable.draw(canvas)
    }
```

Reply:
For me, your suggestion makes it more complex and doesn't add too much value. I use the .apply() magic in other circumstances. BTW, you wrote this code some weeks ago, I just moved it to another class 😛

Reply:
GRAET! I'm improving my Kotlin skills! 💃

Reply:
😂